services/galley/src/Galley/API/Internal.hs

```diff
@@ -194,7 +193,7 @@ sitemap = do
 
   put "/i/teams/:tid/features/search-visibility" (continue Teams.setTeamSearchVisibilityAvailableInternalH) $
     capture "tid"
-      .&. jsonRequest @TeamSearchVisibilityAvailableView
+      .&. jsonRequest @Public.TeamFeatureStatus
```

Comment:
the 3 put end-points for the 3 features could easily be one.  same for internal and public get.

Reply:
(ah, it already happened for the public get.)